ResearchKit/Consent/ORKConsentLearnMoreViewController.m

```diff
@@ -54,11 +54,9 @@ - (instancetype)initWithHTMLContent:(NSString *)content {
     return self;
 }
 
--(instancetype)initWithContentURL:(NSURL * __nonnull)contentURL
-{
+- (instancetype)initWithContentURL:(NSURL * __nonnull)contentURL {
```

Comment:
We don't normally add nullability annotations in implementation files. Could you take the chance to review if there are more around, and remove them?

I'll add this to the *Coding Style Guideline* later.